docs/workloads/azure/data/getting_started/core_data_platform_deployment_azure.md

```diff
@@ -42,7 +38,9 @@ A YAML file containing a template Azure DevOps CI/CD pipeline for building and d
 2. Go to Pipelines, and then select **New pipeline**
 3. Name the new pipeline, e.g. `amido.stacks-data-infrastructure`
 4. For the pipeline definition, specify the YAML file in the project repository feature branch (`air-infrastructure-data.yml`) and save
-5. The new pipeline will require access to any Azure DevOps pipeline variable groups specified in the pipeline YAML. Under each variable group, go to 'Pipeline permissions' and add the pipeline.
+5. Make sure that the Enable Private Networks box is set to your preffered deployment method. Unticking the box will issue a public network deployment.
```

Comment:
There needs to be some sort of information here around what the different deployment methods do. In the text that was there previously there was a link to `../architecture/infrastructure_data_azure#networking`... think it's useful to keep a link to that (may need to review the content on that page is still accurate)?

Reply:
Typo: `preferred`